### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by danielnorkin**
# Description

Wrote out the key logs for quick copy and paste.

## Related Issue

No related issues

## Motivation and Context

It makes it easier for our team to demonstrate the logs

## How Has This Been Tested

yes

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by danielnorkin** [comment]
Sure, I can make it a one liner. Can i simple edit this PR, or just cancel and resubmit?

**Comment by danielnorkin** [comment]
Updated PR as per @bitwiseguy

**Comment by danielnorkin** [comment]
This is still pending a few more approvals

File: radish34/README.md [hunk 1]
```diff
@@ -166,9 +166,9 @@ After you've done that, log in to the Github package registry by running
 7. Run `docker-compose up -d && docker-compose restart`
    - This will reinstate and restart all `radish` containers
    - Wait about 10 seconds to give containers time to complete their initialization routines
-   - Run `docker-compose logs -f {SERVICE_NAME}` to check the logs of specific containers. Key ones to watch are: `api-{role}` and `ui`
+   - Run `docker-compose logs -f {SERVICE_NAME}` to check the logs of specific containers. Key ones to watch are api-{role} and ui. For example: `docker-compose logs -f api-buyer docker-compose logs -f api-supplier1 docker-compose logs -f api-supplier2 docker-compose logs -f ui`
```
Comment (breakpointer) [inline radish34/README.md]: Sorry if I was not clear in my previous comment. I like the concept of having a quick list of logs to look at when running the system however, this line is hard to parse visually (e.g. read) also you can't copy/paste it because it's not valid. 
Please change it to `docker-compose logs -f api-buyer api-supplier1 api-supplier2 ui` This one line tracks all the logs you specified, it is a valid command.

**Comment by danielnorkin** [comment]
My original request, even though it was longer, wrote out each individual
command. Will the command you wrote open the logs in one terminal, or open
seperate terminals displaying those logs?

Maybe putting the reference of all of the logs doesnt belong in the readme,
but in another area? Maybe in the docs? I have it saved locally in a .txt
right now.

File: radish34/README.md (hunk 1 above)
Comment (danielnorkin) [reply radish34/README.md]: My original request, even though it was longer, wrote out each individual command. Will the command you wrote open the logs in one terminal, or open seperate terminals displaying those logs? Maybe putting the reference of all of the logs doesnt belong in the readme, but in another area? Maybe in the docs? I have it saved locally in a .txt right now.

Comment (breakpointer) [reply radish34/README.md]: okay. Now I'm a bit confused. I thought the goal was to write "out the key logs for quick copy and paste". 

The command I wrote tails the logs of the containers specified. You can read about the command here https://docs.docker.com/compose/reference/logs/ It accepts multiple container names. Much like all the other commands in the readme, to use the command you find/open a new terminal window and paste in the command and hit <enter>. The logs from the containers are interlaced but labeled and color coded to make it easy to view their source.

re: "Maybe putting the reference of all of the logs doesnt belong in the readme, but in another area?" I don't understand what you mean here.

Comment (breakpointer) [inline radish34/README.md]: Could you change the example command on this line to `docker-compose logs -f api-buyer api-supplier1 api-supplier2 ui`

Comment (danielnorkin) [reply radish34/README.md]: @breakpointer  - I added it now.
